### PR TITLE
Osaiteh's Pull Request

### PR DESCRIPTION
**Pull request by Osaiteh**

### CODE REVIEW

File: pagecarton/core/application/modules/Application/SiteInfo.php [hunk 1]
```diff
@@ -83,7 +83,7 @@ public function init()
...
             $html .= $settings['site_description'] ? '<br><br><p>' . $settings['site_description'] . '</p>' : null;
-            $html .= self::hasPriviledge( array( 99, 98 ) ) ? '<br><br><p style="font-size:x-small;"><a  style="color:inherit;text-transform:uppercase;" onclick="ayoola.spotLight.showLinkInIFrame( \'' . Ayoola_Application::getUrlPrefix() . '/tools/classplayer/get/object_name/Application_Settings_Editor/settingsname_name/SiteInfo/?' . '&xpc_form_element_whitelist=site_headline,site_description,cover_photo\', \'page_refresh\' );" href="javascript:">[edit site headline and description]</a></p>' : null;
+            $html .= self::hasPriviledge( array( 99, 98 ) ) ? '<br><br><p style="font-size:x-small;"><a  style="color:inherit;text-transform:uppercase;" onclick="ayoola.spotLight.showLinkInIFrame( \'' . Ayoola_Application::getUrlPrefix() . '/tools/classplayer/get/object_name/Application_Settings_Editor/settingsname_name/SiteInfo/?' . '&xpc_form_element_whitelist=site_headline,site_description,cover_photo\', \'page_refresh\' );" href="javascript:">[Give an headline and a description to your site]</a></p>' : null;
```
Comment (ayoolafalola) [inline pagecarton/core/application/modules/Application/SiteInfo.php]: This part is for update. So the phrase "Give an headline and a description to your site" may not be appropriate.

Comment (ayoolafalola) [reply pagecarton/core/application/modules/Application/SiteInfo.php]: What phrase will a user see to denote an "update" to existing record?

Comment (Osaiteh) [reply pagecarton/core/application/modules/Application/SiteInfo.php]: this: Change the site`s Headline and Description

Comment (Osaiteh) [reply pagecarton/core/application/modules/Application/SiteInfo.php]: or: Update the site's headline and description

**Pull request closed without merging**